xbmc/utils/Variant.h

```diff
@@ -87,8 +87,10 @@ class CVariant
   uint64_t asUnsignedInteger(uint64_t fallback = 0u) const;
   uint32_t asUnsignedInteger32(uint32_t fallback = 0u) const;
   bool asBoolean(bool fallback = false) const;
-  std::string asString(const std::string &fallback = "") const;
-  std::wstring asWideString(const std::wstring &fallback = L"") const;
+  std::string asString(const std::string& fallback = "") const&;
```

Comment:
is this the same as `const std::string& asString(const std::string& fallback = "");` ?

Reply:
sorry, should have read the commit message :)